extensions/gubbins/gubbins-db/src/gubbins/db/sql/jobs/schema.py

```diff
@@ -1,17 +1,134 @@
 from diracx.db.sql.job.db import JobDBBase
-from diracx.db.sql.utils import Column
+from diracx.db.sql.job.schema import AccountedFlagEnum, InputData
+from diracx.db.sql.utils import Column, EnumBackedBool, NullColumn
+from diracx.db.sql.utils.types import SmarterDateTime
 from sqlalchemy import (
     ForeignKey,
+    Index,
     Integer,
     String,
+    Text,
 )
 
+# NOTE: We need to remove the original Jobs table from metadata because
+# GubbinsJobs modifies an existing column (job_name size from 128 to 512).
+# Simple inheritance works fine for ADDING columns (see GubbinsInputData below),
+# but MODIFYING existing columns in SQLAlchemy requires replacing the table definition.
```

Comment:
> MODIFYING existing columns

IIRC we had said that we didn't want to support this, do you have a use case in mind?

Reply:
> We explicitly said we would not do that because it has too many implications. We will even have to modify some LHCb database to follow that.

Is that written anywhere in the documentation?
I found that block of documentation about DB extensions, but it's not explicitly mentioned: https://diracx.diracgrid.org/en/latest/dev/explanations/extensions/#extended-db

If not written anywhere, I think we should clearly mention it there (this section of the documentation is probably the right place)

(In the future, as we discussed, this should likely go through a Github Issue for discussion first)